02-oop-inheritance/keypoints/2_7-12/script.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Created by MaximilianoDaniel on 17/09/2015.
+ */
+
+var inheritPrototype = function (childClass, parentClass){
+
+    var copyOfParent = Object.create(parentClass.prototype);
+
+    //Is necessary to point to the childClass constructor
+    copyOfParent.constructor = childClass;
+    childClass.prototype = copyOfParent;
+};
+
+function augment( receivingClass, givingClass ) {
+
+    // only provide certain methods
+    // the index starts at two because the first and second arguments are receivingClass and givingClass
+    if ( arguments[2] ) {
```

Comment:
Exactly, the third argument would be located in the third position of your array (second index), and that's what you're checking to know if you have to iterate the rest of the arguments (meaning that you're leaving ``receivingClass`` and ``givingClass`` behind).
This code is correct, I was just suggesting using other kind of check that, under my criteria, could be easier to read. ;)